### ISSUE TITLE
Upload app to iTunes Connect

### ISSUE THREAD

**Issue by wvdk** [wvdk/Life-Calendar#43]
I'm thinking we should do this earlier rather than later because I've never submitted a Mac app before. There might be some unexpected complications in the code signing / submission process.

**Comment by wvdk** [comment]
One thing that must happen before this is some solidification of thought about how to handle version numbering.

**Comment by wvdk** [comment]
- [ ] Registar new bundle ID (vdk.wes.life-calendar)

**Comment by wvdk** [comment]
BLOCKER: I cannot upload with a beta build of Xcode. This will have to be put on hold until Xcode 9 officially hits the shelves.
